pkgs/applications/video/kodi/unwrapped.nix

```diff
@@ -110,7 +111,15 @@ in stdenv.mkDerivation {
     version = kodiVersion;
 
     src = kodi_src;
-
+    patches = [
+      # Fix compatiblity with fmt 10.0 (from spdlog).
+      # Remove with the next release: https://github.com/xbmc/xbmc/pull/23453
+      (fetchpatch {
+        name = "Fix fmt10 compat";
+        url = "https://github.com/xbmc/xbmc/pull/23453.patch";
```

Comment:
This URL will be 404 if the pull request will change in the future. If the PR was merged, you could have used the merge commit as the commit hash for the URL. But, since it is not yet merged, I don't think you have a choice but to put each of these commits in with it's own `fetchpatch`.

Reply:
won't be 404 but the checksum will fail, yes. As said above; this is no longer urgent let's wait for the PR to be merged; I'll mark this as draft until then

Reply:
https://github.com/xbmc/xbmc/pull/23453 has been merged.

Reply:
Thanks! I was following the PR and rebased this when it was merged, but build just takes too long on this machine and I forgot to push/update this as I wasn't around when it finished...

I've re-checked now, everything is still in order; the pull/xx.patch file should now be stable to use as is without referring to individual commits.